src/helpers/subscribe.ts

```diff
@@ -0,0 +1,16 @@
+export async function subscribeToNewsletter(email: string) {
+  const res = await fetch('https://proxy.landing.rarimo.com/v1/subscriptions', {
```

Comment:
Please check that there are no issues with CORS

Reply:
works fine on staging & production, there's `Access-Control-Allow-Origin: *` header in the response